content/style-injector.js

```diff
@@ -30,12 +32,28 @@ function createStyleInjector({compare, setStyleContent, onUpdate}) {
     createStyle
   };
 
+  function _initCreationDoc() {
+    // FF59+ workaround: allow the page to read our sheets, https://github.com/openstyles/stylus/issues/461
+    // Since browser version can be spoofed, checking for getPreventDefault that was removed in FF59
+    const pageContextDoc = !Event.prototype.getPreventDefault && document.wrappedJSObject;
+    creationDoc = pageContextDoc || document;
```

Comment:
This can be rewritten as:
```js
const pageContextDoc = ...
let pageStyleApplied = false;
if (pageContextDoc) {
  ...
}
createionDoc = pageStyleApplied ? pageContextDoc : document;
```

Reply:
Why? Just because it's more verbose doesn't make it easier to understand. For example I don't understand why an extra variable is used. The code would flow in two "streams" so to understand it you need to follow both. As Occam said, "Entities should not be multiplied without necessity".

Reply:
I prefer to avoid assigning `createionDoc` in multiple places. `createionDoc = pageStyleApplied ? pageContextDoc : document;` looks clearer to me. Though I guess it is not a big deal here since the function is pretty short.

It doesn't add an extra variable. I renamed `applied` to `pageStyleApplied`.

Reply:
creationDoc needed to be set before `createStyle` so your code wouldn't work as is, but anyway I've reworked it now so hopefully it's better.